docs/maps/connect-to-ems.asciidoc

```diff
@@ -19,5 +19,5 @@ behind a firewall. If this happens, you can disable the EMS connection to avoid
 To disable EMS, change your <<settings, kibana.yml>> file.
 
 . Set `map.includeElasticMapsService` to `false` to turn off the EMS connection.
-. Set `map.tilemap.url` to the URL of your tile server. This configures the **Maps** default tile layer.
+. Set `map.tilemap.url` to the URL of your tile server. This configures **Elastic Maps** default tile layer.
```

Comment:
Suggest reversing this sentence: This configures the default tile layer of Elastic Maps.